nats_test.go

```diff
@@ -137,7 +137,10 @@ func TestExpandPath(t *testing.T) {
 			{path: "/Foo/Bar", userProfile: `C:\Foo\Bar`, wantPath: "/Foo/Bar"},
 			{path: "Foo/Bar", userProfile: `C:\Foo\Bar`, wantPath: "Foo/Bar"},
 			{path: "~/Fizz", userProfile: `C:\Foo\Bar`, wantPath: `C:\Foo\Bar\Fizz`},
-			{path: `${HOMEDRIVE}${HOMEPATH}\Fizz`, userProfile: `C:\Foo\Bar`, wantPath: `C:\Foo\Bar\Fizz`},
+			// That one would fail because expandPath(), if not finding `~` returns
+			// the given path, which since ${HOMEDRIVE}${HOMEPATH} is not set,
+			// would return `\Fizz` but test expects `C:\Foo\Bar\Fizz`?
+			// {path: `${HOMEDRIVE}${HOMEPATH}\Fizz`, userProfile: `C:\Foo\Bar`, wantPath: `C:\Foo\Bar\Fizz`},
```

Comment:
@variadico I had to comment out this line specific test:
```
{path: `${HOMEDRIVE}${HOMEPATH}\Fizz`, userProfile: `C:\Foo\Bar`, wantPath: `C:\Foo\Bar\Fizz`},
```
My reading of the code, since there is no `~`, expandPath() would simply returned what is given, in this case `\Fizz` since the 2 env variables are not set, and so the test would fail.
Could you have a look at what was the intent or if this need tweaking?

Reply:
I think perhaps I meant
```
wantPath: `\Fizz`
```
But, it should probably be ok to remove this case.

If `${HOMEDRIVE}${HOMEPATH}` are set, then the correct path will be returned. Otherwise, the user will get an error and they'll know to give us a different path. Probably a path with `${USERPROFILE}` so we can expand that.

Reply:
> If ${HOMEDRIVE}${HOMEPATH} are set, then the correct path will be returned.

But that's the thing, in this test's config, they are not. So should I remove completely or do you want to keep it with `wantPath: "\Fizz"`?

Reply:
Ok, as agreed on Slack, just removing this line.